### PR TITLE
CLI: Seclude version output functionality out of CLI class

### PR DESCRIPTION
**Pull request by medikoo**
Addresses 1.0 from https://github.com/serverless/serverless/issues/8364

### CODE REVIEW

File: lib/cli/eventually-list-version.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  process.stdout.write(
+    `Framework Core: ${version}${installationModePostfix}\n` +
+      `Plugin: ${dashboardPluginVersion}\n` +
+      `SDK: ${sdkVersion}\n`
+  );
+
+  const userNodeVersion = Number(process.version.split('.')[0].slice(1));
+  // only show components version if user is running Node 8+
+  if (userNodeVersion >= 8) {
```
Comment (pgrzesik) [inline lib/cli/eventually-list-version.js:38]: I believe this check is redundant as we require `node>=10` now

**Review by pgrzesik — CHANGES_REQUESTED** [review]
One note about node version check, otherwise it looks good :+1:

**Review by pgrzesik — APPROVED** [review]
:+1:

File: lib/cli/eventually-list-version.js (hunk 1 above)
Comment (medikoo) [reply lib/cli/eventually-list-version.js]: Good find, thanks for pointing!

**Pull request merged**